R/pkg/R/DataFrame.R

```diff
@@ -3444,7 +3445,8 @@ setMethod("as.data.frame",
 #' @note attach since 1.6.0
 setMethod("attach",
           signature(what = "SparkDataFrame"),
-          function(what, pos = 2L, name = deparse(substitute(what), backtick = FALSE),
+          function(what, pos = 2L,
+                   name = paste(deparse(substitute(what), backtick = FALSE), collapse = " "),
```

Comment:
Sorry for an ignorant question. Why do we need `collapse = " "`?

Reply:
Sorry I think I responded to this in the main conversation thread rather than here. Resolving.